Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -698,35 +704,68 @@ def get_file_data(self, file_hash):
         cmd_url = f'/files/{file_hash}'
         return self.ms_client.http_request(method='GET', url_suffix=cmd_url)
 
-    def list_indicators(self, indicator_id: Optional[str] = None, page_size: str = '50', limit: int = 50) -> List:
+    def sc_list_indicators(self, indicator_id: Optional[str] = None, limit: int = 50,
+                           should_use_security_center=False) -> List:
+        """Lists indicators. if indicator_id supplied, will get only that indicator.
+
+                Args:
+                    indicator_id: if provided, will get only this specific id.
+                    limit: Limit the returned results.
+                    should_use_security_center: whether to use the security center's scope and resource.
+
+                Returns:
+                    List of responses.
+                """
+        results = {}
+        # For getting one indicator
+        cmd_url = urljoin(SECURITY_CENTER_INDICATOR_ENDPOINT,
+                          indicator_id) if indicator_id else SECURITY_CENTER_INDICATOR_ENDPOINT
+
+        params = {'$top': limit}
+        resp = self.indicators_http_request(should_use_security_center,
+                                            'GET', full_url=cmd_url, url_suffix=None, params=params, timeout=1000,
+                                            ok_codes=(200, 204, 206, 404), resp_type='response')
+        # 404 - No indicators found, an empty list.
+        if resp.status_code == 404:
+            return []
+        resp = resp.json()
+        results.update(resp)
+
```

Comment:
we are missing here the pagination handling and removal of the `@odata.context` that the current command has